README.md

```diff
@@ -10,7 +10,15 @@ Information about our deployments can be found on our [Google Drive document](ht
 
 ### The demoservice app
 
-#### The easy way (Docker - Untested)
+#### Docker Compose
+
+A close version to the production set up can be started using docker compose. All services are defined under docker-compose.yml in the project root.
+
+``` bash
+docker-compose up
+```
+
```

Comment:
Could you please add a line here mentioning where the server will be running (http://0.0.0.0:8000 I guess).

Although, if you want to be thorough, it would be good [to nominate and use a unique port for this project](https://github.com/canonical-webteam/practices/blob/master/local-development/ports.md).

Reply:
Updated this, running at 8099. I'll updated that document after this is merged.

Reply:
I think it was just the case that PostgreSQL was not actually yet ready to receive connections.
The depends_on config in the YAML only waits until the container is running before starting other containers so it might be that your web container was ready before the DB. Does it not work if you try and access the web UI?

Reply:
Updated to use Sqlite too. That should fix the database issue you were seeing.